src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -278,25 +278,37 @@ public override void Dispose()
             this.logger.LogInformation("Flushing peers...");
             this.flushAddressManagerLoop.Dispose();
 
+            this.logger.LogInformation("peerAddressManager...");
```

Comment:
You probably want to add `Disposing bla...`

Reply:
I did, as it's in the Dispose() method this is shown in the output.  It would say "... Dispose() Disposing peerAddressManager..."

Reply:
I'll add Disposing back in and we can see what it looks like.